src/zenml/integrations/bentoml/__init__.py

```diff
@@ -32,7 +32,7 @@ class BentoMLIntegration(Integration):
 
     NAME = BENTOML
     REQUIREMENTS = [
-        "bentoml>=0.13",
+        "bentoml==0.13.2",
```

Comment:
This should be bentoml>=1.0.0

Reply:
@aarnphm Yes, I just had to do that because some of the requirements of bentoML clash with other integration dependencies (Airflow integration, and Seldon Core integration) and I want to see if all GH would pass then force it to greater than 1.0.0